BUILD

```diff
@@ -12,33 +12,7 @@ bzl_library(
     srcs = [
         ":defs.bzl",
         ":specs.bzl",
-        "//private:artifact_utilities.bzl",
-        "//private:constants.bzl",
-        "//private:coursier_utilities.bzl",
-        "//private:dependency_tree_parser.bzl",
-        "//private:java_utilities.bzl",
-        "//private:proxy.bzl",
-        "//private:versions.bzl",
-        "//private/lib:bzlmod.bzl",
-        "//private/rules:artifact.bzl",
-        "//private/rules:coursier.bzl",
-        "//private/rules:generate_pin_repository.bzl",
-        "//private/rules:has_maven_deps.bzl",
-        "//private/rules:java_export.bzl",
-        "//private/rules:javadoc.bzl",
-        "//private/rules:jvm_import.bzl",
-        "//private/rules:maven_bom.bzl",
-        "//private/rules:maven_bom_fragment.bzl",
-        "//private/rules:maven_install.bzl",
-        "//private/rules:maven_project_jar.bzl",
-        "//private/rules:maven_publish.bzl",
-        "//private/rules:maven_utils.bzl",
-        "//private/rules:pin_dependencies.bzl",
-        "//private/rules:pom_file.bzl",
-        "//private/rules:urls.bzl",
-        "//private/rules:v1_lock_file.bzl",
-        "//private/rules:v2_lock_file.bzl",
-        "//settings:stamp_manifest.bzl",
+        "@rules_license//:docs_deps",
```

Comment:
Sigh. The circular loop of skylib StarlarkLibraryInfo being needed in a rule set that wants not deps is a problem. This is a case where bazel needs a conditional.

What I need is a module extension to detect that skylib is loaded already, and if so, create a local repo that provide bzl_library, either pointing to the real one or a fake one.

That makes more sense in my head than it probably does on paper.